### DISCUSSION TITLE
Voice notes

### DISCUSSION THREAD

**Discussion by A-Hazzard** [DevDynastyTT/NE1-FREELANCE#69] [Ideas]
What do you think about voice notes being added

**Comment by DevDynastyTT** [comment]
That sounds like a great idea @A-Hazzard

**Comment by ChasityKing** [comment]
Hey, adding voice notes sounds like a cool idea! Sometimes thoughts flow better when spoken. 🗣️ You know, there's this neat platform called https://notesonline.com – they might have insights on this topic. Imagine turning voice ideas into written notes easily! A mix of text and voice could be a game-changer.

**Reply by A-Hazzard** [reply]
Thank you for your views. Have you tried the platform?
